base/client.jl

```diff
@@ -393,8 +392,7 @@ function load_InteractiveUtils(mod::Module=Main)
     if !isdefined(mod, :InteractiveUtils)
         try
             let InteractiveUtils = require(PkgId(UUID(0xb77e0a4c_d291_57a0_90e8_8db25a27a240), "InteractiveUtils"))
-                Core.eval(mod, :(const InteractiveUtils = $InteractiveUtils))
-                Core.eval(mod, :(using .InteractiveUtils))
+                Core.eval(mod, :(using InteractiveUtils))
```

Comment:
Won't this be broken if the user didn't have the right name in their load path / Manifest?

Reply:
Typically these are in the sysimage; if you've removed them I'm not sure we care about trying to handle that for you.

Reply:
We do probably care though (for `-i`) because we are going to remove them ASAP, and this already was working. This is not about loading them, but about correctly mapping the name to the PkgId

Reply:
Ah yes I see --- I think the solution is to put the explicit identifier somewhere else, maybe in MainInclude?

Reply:
What is the goal here? I seem to see the same tab-completion behavior for `REPL` (if I call `using REPL`) and for `InteractiveUtils`

Reply:
It's just cosmetic: avoid tab-completing `Main` to `MainInclude`, and remove `InteractiveUtils` from `varinfo()` output.